vagrant/README.md

```diff
@@ -2,30 +2,44 @@ Developing using Docker + Vagrant
 ---------------------------------
 
 The current tooling here is oriented towards
-doing builds inside a CentOS 7 pet container,
+doing builds inside a Fedora 30 pet container,
 with Vagrant on the host.
 
-You should share the git working directory with the c7 container.
+You should share the git working directory with the f30 container.
 Assuming you have git repositories stored in `/srv`, something like:
 
 ```
-docker run --name c7dev --privileged -v /srv:/srv --net=host -ti centos bash
+docker run --name f30dev --privileged -v /srv:/srv --net=host -ti registry.fedoraproject.org/fedora:30 bash
```

Comment:
Hmm, doing f29 development in an f30 container? That's likely to cause issues. In general, the build environment should match the target env.

Reply:
Yeah - just tried this again following these steps (got shared object issues). I wonder if somehow I had tested this with an F29 container thinking it was 30. Switching this to F29 (just testing with that now)!